COPYRIGHT.md

```
@@ -1,4 +1,5 @@
-Copyright © 2025 The Stable Order <br />
+# Copyright
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)